src/config.ts

```diff
@@ -283,7 +286,7 @@ export class ConfigurationReader implements vscode.Disposable {
     get buildToolArgs(): string[] {
         return this.configData.buildToolArgs;
     }
-    get parallelJobs(): number {
+    get parallelJobs(): number | undefined {
         return this.configData.parallelJobs;
```

Comment:
`this.configData.parallelJobs` will always be a number.  You should do the inspect of the setting here so that returning `undefined` is possible.